discovery-infra/test_infra/controllers/node_controllers/host.py

```diff
@@ -0,0 +1,68 @@
+import logging
+from test_infra.controllers.node_controllers import ssh
+
+
+class Host:
```

Comment:
Can be a `dataclass`

Reply:
what it gives?

Reply:
i don't see

Reply:
Just that you won't need to declare a constructor.

Reply:
u still will need to provide params. easier for me this way